### PR TITLE
Feature removal - office notes

### PR DESCRIPTION
**Pull request by alpinus4**
Deleted _office notes_ files.
Fixes: https://github.com/LibreHealthIO/LibreEHR/issues/935

### CODE REVIEW

**Comment by aethelwulffe** [comment]
This is just a feature removal, right?

**Comment by alpinus4** [comment]
Yea

File: library/onotes.inc [hunk 1]
```diff
@@ -1,76 +0,0 @@
...
-	return sqlQuery("select $cols from onotes where id=? order by date DESC limit 0,1", array($id));
-}
-
-function getOnoteByDate ($date, $activity = "1", $cols = "*", $limit="all", $offset="0")
-{
-
-        $sqlBindArray = array();
-	if($limit == "all")
-	{
-		$sql = "select $cols from onotes where date like ? ";
```
Comment (aethelwulffe) [inline library/onotes.inc]: Remove onotes from database.sql

**Comment by tmccormi** [comment]
Definitely dump Onotes entirely.

**Comment by tmccormi** [comment]
I replaced it entirely for one customer with a mini forum tool..

phpminiforum-2
phpminiforum

That tool: my little forum - http://mylittleforum.net/  is not accessible now, but it was GPL 3 and I have the code.

**Comment by teryhill** [comment]
PM me a link to it if you will.

File: library/onotes.inc (hunk 1 above)
Comment (teryhill) [reply library/onotes.inc]: Also add a removal of it from existing databases in the 1_0_1-to-2_0_0_upgrade.sql

Remove it from the menu also

**Comment by alpinus4** [comment]
I left onotes into menu.sql ,because wouldn't  want to destroy anything.... 
But ... I was thinking about removing the 43rd and 70th lines.. Would it be good idea?

**Comment by aethelwulffe** [comment]
I fixed conflicts in this branch, and deleted office notes from the menu.  That happened when master was merged in.  This is ready to go from my perspective.  We have a lot of database.sql cleanup to do, and any missing pieces should be gone over in that long awaited piece of partisan code.

**Comment by aethelwulffe** [comment]
We are getting a LOT of conflicts from that file.  The issue with removal of the fields with old databases (and face it, there are not a lot), is that there might be data that could be lost.  There isn't anywhere to migrate it to, so I say leave as is.

**Pull request merged**